code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,22 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
```

Comment:
@Krausus the most recent commit as of this comment performs checks to make sure the box is filled with crayons rather than photocopys of someones butt, or worst, nothing! It doesn't force you to have one of each basic color crayon, but that should not be an issue.